config/smarty.config.inc.php

```diff
@@ -177,6 +177,10 @@ function smartyCleanHtml($data)
 
 function smartyClassname($classname)
 {
+    if (!is_string($classname)) {
```

Comment:
Why not adding a `string` type to the parameter ?

Reply:
Wooups I thought you would tell me "no it's a BC break" 🤣